ci/jenkins/test.sh

```diff
@@ -309,12 +310,18 @@ function deliver_antrea {
     echo "---" >> build/yamls/antrea.yml
     cat build/yamls/antrea-prometheus.yml >> build/yamls/antrea.yml
 
+    # Increase verbose for flow aggregator dual stack e2e test
+    sed -i "s|--v=0|--v=4|g" build/yamls/antrea.yml
```

Comment:
does this need to be checked in? I presume this was done for debugging.

Reply:
Thanks, addressed.